index.html

```diff
@@ -215,18 +227,21 @@ <h3 class="intro">About <br>Myself</h3>
                         <span class="lang-head">Languages</span> <img src="./images/Chevron-Down.png" alt="Chevron-Down"/>
                     </a> 
                 </li>
-                <li class="about-lang">
-                    <img src="./images/javaScript-icon.png" alt="JavaScript"/>
-                    <a href="#" class="lang javascript">JavaScript</a> 
-                </li>
-                <li class="about-lang">
-                    <img src="./images/html-icon.png" alt="Html"/>
-                    <a href="#" class="lang html">HTML</a> 
-                </li>
-                <li class="about-lang">
-                    <img src="./images/css-icon.png" alt="CSS"/>
-                    <a href="#" class="lang css">CSS</a> 
+                <li class="about-lang-wrapper">
+                    <li class="about-lang">
+                        <img src="./images/javaScript-icon.png" alt="JavaScript"/>
+                        <a href="#" class="lang javascript">JavaScript</a> 
+                    </li>
+                    <li class="about-lang">
+                        <img src="./images/html-icon.png" alt="Html"/>
+                        <a href="#" class="lang html">HTML</a> 
+                    </li>
+                    <li class="about-lang">
+                        <img src="./images/css-icon.png" alt="CSS"/>
+                        <a href="#" class="lang css">CSS</a> 
+                    </li>
                 </li>
+                
                 <li class="framework-skills">
                     <a href="#" class="framework">Framework</a>
                     <img src="./images/Chevron-Right.png" alt="Right-caret"/>
```

Comment:
- [ ] As this about section is not similar to the Figma design kindly implement all the changes like color, size, font-family, font-weight
### your design
![hiiiiiiiii](https://user-images.githubusercontent.com/81727988/171617437-97fef1ce-909a-4812-8abb-347ccc26d500.png)
### Figma design
![jjjj](https://user-images.githubusercontent.com/81727988/171617479-3b2dbab4-8fa7-4132-b95e-9f6e7a32d327.png)

Reply:
@principlebrothers Hello, I would suggest you make the js, HTML, and CSS inline in the same line and use the font family provide by the microverse. You need to work on styling.